packages/sdk-react/src/ui/FileViewer/FileViewer.tsx

```diff
@@ -138,14 +188,14 @@ const FileViewerComponent = ({
 
   const isPdf = mimetype === 'application/pdf';
 
+  if (!url) {
+    return <ErrorComponent onError={onReloadCallback} />;
+  }
+
   const renderFile = () => {
     if (isPdf)
       return (
-        <Document
-          file={url}
-          onLoadSuccess={onDocumentLoadSuccess}
-          onLoadError={console.error} //TODO add error view when design will be ready
-        >
+        <Document file={url} onLoadSuccess={onDocumentLoadSuccess}>
```

Comment:
Shouldn't we display some error message using `onLoadError` props if the document has some problems?

Reply:
This is a quite strange library :) It doesn't call any `onLoadError`, `onSourceError`, or any other callbacks if there is no `url` has been specified ([based on react-pdf documentation](https://github.com/wojtekmaj/react-pdf?tab=readme-ov-file#props)).
So the only thing is to provide `noData` but this is when we have no file by provided `url` so I've checked `url` before sending any request to remove any PDF controls that we have right now.
So that's how it looks like:
![Screenshot 2024-05-24 at 15 40 45](https://github.com/team-monite/monite-sdk/assets/6944144/d3eb02fa-61f6-4adf-b7eb-1d82d5ca304a)

So I could provide `noData` "just for safety" if our BE returns 200 with `url` but without any PDF by that link (I don't think that it's a real case but I have no problem with that check either)

Commit: [226b339](https://github.com/team-monite/monite-sdk/pull/71/commits/226b339db677840fd716a6395a4a35062e204cf7)

Reply:
Thanks for the clarification!